### DISCUSSION TITLE
Maui popup selectable label text

### DISCUSSION THREAD

**Discussion by ehaviv** [CommunityToolkit/Maui#1611] [Q&A]
I tried to enable selectable label text in the popup in the same way l did in app mainpage . but that's not works
is it possible to fixing this.
Thank you.

**Comment by bijington** [comment]
What do you mean by a selectable Label? If you have what you believe is a bug can you please open an issue with the required information?

**Comment by ehaviv** [comment]
In selectable I mean to allow me to
copy text of the popup label for the purpose of pasting it somewhere.

Thank you.


בתאריך יום ג׳, 19 בדצמ׳ 2023, 11:13, מאת Shaun Lawrence ‏<
***@***.***>:

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
